pkg/envoy/proxy.go

```diff
@@ -343,40 +344,17 @@ func writeBootstrapConfig(cfg []byte) (string, func() error, error) {
 		os.TempDir(),
 		fmt.Sprintf("envoy-%x-bootstrap.json", time.Now().UnixNano()+int64(os.Getpid())),
 	)
-	if err := syscall.Mkfifo(path, 0600); err != nil {
-		return "", nil, err
-	}
-
-	// O_WRONLY causes OpenFile to block until there's a reader (Envoy). Opening
-	// the pipe with O_RDWR wouldn't block but would result in just sending stuff
-	// to ourself.
-	//
-	// TODO(boxofrad): We don't have a way to cancel this goroutine. If the Envoy
-	// process never opens the other end of the pipe this will hang forever. The
-	// workaround we use in `consul connect envoy` is to write to the pipe in a
-	// subprocess that self-destructs after 10 minutes.
```

Comment:
We should add a changelog entry for this in `.changelog/`

Reply:
done